src/components/command-bar/commands/search/unified-search/components/unified-hit/helpers/build-url-path.ts

```diff
@@ -0,0 +1,48 @@
+import { getTutorialSlug } from 'views/collection-view/helpers'
+import { getIntegrationUrl } from 'lib/integrations'
+// Types
+import type { Hit } from 'instantsearch.js'
+
+/**
+ * Builds a URL path to an arbitrary hit from our unified `<env>_DEVDOT_omni`
+ * Algolia indices.
+ *
+ * TODO: consider baking a `urlPath` property into each Algolia object?
+ */
+export function buildUrlPath(searchHit: Hit): string {
```

Comment:
Noteworthy: I've been considering the value of baking the `urlPath` for each item into the Algolia search object. So far leaning towards _not_ doing that. I think the effort of refining the search card will confirm that it's feasible to build a highlighted `urlPath` rather than bake it in, but if not (or if the "construct URL from disparate ranked properties" approach has weird ranking implications), might mosey back to the back-end and reconsider.

Reply:
I am slightly on the fence as it sounds like you are on this one, but I think it's probably the right move to not hard code the URL into the indexing!